tss-esapi/src/structures/algorithm/symmetric/sensitive_create.rs

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use crate::{
+    structures::{Auth, SensitiveData},
+    traits::{Marshall, UnMarshall},
+    tss2_esys::{TPM2B_SENSITIVE_CREATE, TPMS_SENSITIVE_CREATE},
+    Error, Result, ReturnCode, WrapperErrorKind,
+};
+use log::error;
+use std::convert::{TryFrom, TryInto};
+/// Structure that defines the calues to be placed in the sensitive
+/// area of a created object.
+///
+/// # Details
+/// This corresponds to the TPMS_SENSITIVE_CREATE
+/// structure.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct SensitiveCreate {
+    user_auth: Auth,
+    data: SensitiveData,
```

Comment:
(Not something specific to this PR) We should:

1. Implement `Zeroize` on this too
2. Add some documentation, even if a bit "robotic" to these buffer types, at the moment you look at them and struggle to figure out what they're even supposed to be.

Reply:
I had to change the comment that SensitiveCreateBuffer corresponds to the TPM2B type ... it actually does not.

Reply:
Fixed